guide/how-it-works/wallet-privacy.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: guide
+title: Wallet privacy
+description: An overview of how to help users maintain their financial privacy while using a bitcoin daily spending wallet.
+nav_order: 9
+parent: How it works
+permalink: /guide/how-it-works/wallet-privacy/
+main_classes: -no-top-padding
+image: /assets/images/guide/how-it-works/wallet-privacy/wallet-privacy-preview.jpg
+---
+
+{% include picture.html
+    image = "/assets/images/guide/how-it-works/wallet-privacy/wallet-privacy.jpg"
+    retina = "/assets/images/guide/how-it-works/wallet-privacy/wallet-privacy@2x.jpg"
+    mobile = "/assets/images/guide/how-it-works/wallet-privacy/wallet-privacy-mobile.jpg"
+    mobileRetina = "/assets/images/how-it-works/wallet-privacy/wallet-privacy-mobile@2x.jpg"
+    alt-text = "Illustration of an eye, with a cross on top of an ink blot background"
+    width = 1600
+    height = 600
+    layout = "full-width"
+%}
+
+<!--
+This page should inform about what information is made public when sending or receiving, how the application can help minimize unnecessary privacy leaks, basic application functionality to help, and when we can, more advanced options.
+
+- Single-use addresses (avoiding address reuse)
+- Coin selection / labelling
+- Coin join / Pay join
+
+@TODO: address reuse / there should be something in here about also mindfully not giving the same unused address to different persons
+@TODO: address reuse / write glossary term about Gap limit
+-->
+
+# Wallet privacy
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+Privacy is about the ability to control what information is available to others, and what they can do with it. On this page, we will provide a soft introduction to the topic, with a focus on what product builders and users can do to improve their payment privacy.
+
+#### Design with privacy in mind
```

Comment:
This small title looks out of place, why not use a bigger one?

Reply:
Had that initially, but that looked out of place to me. Also tried removing it, but felt like the text below needs a header.